{"tests": 3, "errors": 0, "results": [{"skipped_msg": null, "error_trace": null, "test_method_name": "test_item_not_found", "failure_trace": null, "test_class_name": "GetLoadBalancersTests", "result": "PASSED"}, {"skipped_msg": null, "error_trace": null, "test_method_name": "test_method_not_allowed", "failure_trace": null, "test_class_name": "GetLoadBalancersTests", "result": "PASSED"}, {"skipped_msg": null, "error_trace": null, "test_method_name": "test_get_load_balancers", "failure_trace": "Traceback (most recent call last):\n  File \"/home/server/.virtualenvs/cloud_cafe/lib/python2.7/site-packages/cloudroast/getloadbalancers/smoke/test_run.py\", line 17, in test_get_load_balancers\n    self.assertEquals(r.status_code, 200)\nAssertionError: 401 != 200\n", "test_class_name": "GetLoadBalancersTests", "result": "FAILED"}], "skips": 0, "time": "7.92652106285", "failures": 1}